src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,52 @@
+package view;
+
+import domain.ladder.common.Direction;
+import domain.player.Name;
+
+import java.util.EnumMap;
+import java.util.List;
+
+public class OutputView {
+    private static final EnumMap<Direction, String> directionSymbols = initializedDirectionSymbol();
+
+    private static final String RIGHT_DIRECTION_SYMBOL = "|---";
+    private static final String LEFT_DIRECTION_SYMBOL = "---|      ";
+    private static final String DOWN_DIRECTION_SYMBOL = "|      ";
+    private static final String BLANK_SPACE = "      ";
+    private static final int NAME_SPACE_SIZE = 7;
+    private static final String NAME_SPACE_UNIT = " ";
+
+    public static void printPlayerNames(List<Name> playerNames) {
+        playerNames.stream()
+                   .map(Name::getValue)
+                   .map(OutputView::padString)
+                   .forEach(System.out::print);
+        printNewLine();
+    }
+    private static String padString(String name) {
+        return NAME_SPACE_UNIT.repeat(NAME_SPACE_SIZE - name.length()) + name;
+    }
+    private static void printNewLine() {
+        System.out.print(System.lineSeparator());
+    }
+    public static void printDirections(List<Direction> directions) {
+
+        StringBuilder resultStringBuilder = new StringBuilder();
+        resultStringBuilder.append(BLANK_SPACE);
+
+        directions.forEach(direction -> {
+            String symbol = directionSymbols.get(direction);
+            resultStringBuilder.append(symbol);
+        });
+
+        System.out.println(resultStringBuilder);
+    }
+
+    private static EnumMap<Direction, String> initializedDirectionSymbol() {
```

Comment:
EnumMap 사용 👍 👍 

`directionSymbols`에서는 `Map`으로 추상화해서 들고 있어도 될 것 같은데 구현체를 타입으로 지정한 이유가 있을까요?

Reply:
Enum 타입을 다룰 때, 실행의 안정성과 EnumMap을 사용함으로써 가질 수 있는 코드의 유지 보수성을 생각하여 적용해봤습니다!

Reply:
```suggestion
    private static Map<Direction, String> initializedDirectionSymbol() {
```
을 이야기한 것이었어요. `ArrayList`가 아닌 `List` 인터페이스를 사용하고 계신 것처럼요. 아직 추상화의 개념이 어색하시다면 이번기히에 예시가 나왔으니 한번 공부해보시는것도 좋을 것 같아요.

Reply:
넵 말씀해주신 부분 따로 학습해보도록 하겠습니다!